services/app/src/static/style/chess-pieces.css

```diff
@@ -0,0 +1,99 @@
+/*-----Game Footer-----*/

```

Comment:
These are `GameFooter` styles, lets move those to `gameFooter/styles.css` to have everything in one place on the folder structure.